src/main/flight/pid_init.c

```diff
@@ -292,6 +296,8 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         pidRuntime.pidCoefficient[FD_YAW].Ki *= 2.5f;
     }
     pidRuntime.levelGain = pidProfile->pid[PID_LEVEL].P / 10.0f;
+    pidRuntime.levelFeedforwardGain = pidProfile->pid[PID_LEVEL].F / 100.0f;
```

Comment:
Even with default level F of 50, this gain factor seems too high for most quads.

Based on a fair bit of testing, I'd suggest  `/300.0f`.

If we use this, a default F of 50 is OK on its own, a value of 100 becomes usable with expo of 30-40, and a value of about 150-200 tends to overshoot and is only usable with expo around 60-70.

Reply:
> Here is a table of CLI values relevant to angle_mode, with suggested re-naming for consistency.
> 
> The intent is that when the user types 'get level_' they will find all these values
> 
> Current name	Suggested name	Notes
> angle_level_strength	level_strength	only angle mode
> level_limit	level_angle_limit	only angle mode
> level_gyro_feedforward_adjust	level_gyro_feedforward	only angle mode, adjust seems unnecessary
> angle_level_feedforward	level_feedforward	only angle mode
> pitch_level_expo	level_expo_pitch	angle and horizon
> roll_level_expo	level_expo_roll	angle and horizon
> level_race_mode	level_race_mode	only angle mode

What is this level mode that everyone keeps talking about! 😉 We have ANGLE mode HORIZON mode and ACRO mode. Level mode doesn't exist and why would anyone search for it!!??

Reply:
I'm not super keen to change that scaling as it actually means something at the moment. With a value of 100% the full stick rate is passed to the gyro loop. If you request 10deg/s angle change the feedforward term will pass 10deg/s to the gyro loop.